ui/src/common/QualityInfo.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Chip from '@material-ui/core/Chip'
+import { LOSSLESS_FORMATS } from '../consts'
+
+export const QualityInfo = ({ record, size, ...rest }) => {
+  /* let suffix = 'NO_SUFFIX'
```

Comment:
Actually, can you please remove this commented code?

Reply:
Oh forgot that.